src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCounter = 0;
+
+  const getSum = (num) => {
+    callCounter++;
+
+    return (operator) => {
+      if (callCounter > 3 && callCounter % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return num + operator;
```

Comment:
[CODE STYLE] - Don't nest ternary operators. You're using an if statement inside a returned function which is okay, but make sure to avoid nesting ternary operators as per the code style guide.